digdag-ui/console.jsx

```diff
@@ -642,6 +642,48 @@ class ProjectsView extends React.Component {
   }
 }
 
+class StatusFilter extends React.Component {
+  state = {
+    selectedStatus: 'All'
+  };
+
+  filterSessionsByStatus(sessions, selectedStatus) {
+    switch (selectedStatus) {
+      case 'Success':
```

Comment:
this cases should use a common constant with the rendering function, so strings can't go out of sync (also we should add a type so flow can ensure consitency)